### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ebocher**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # OrbisGIS [![Build Status](https://travis-ci.org/orbisgis/orbisgis.png?branch=master)](https://travis-ci.org/orbisgis/orbisgis)
 
 
-[OrbisGIS] (http://orbisgis.org/) is a cross-platform open-source Geographic Information System ([GIS](https://en.wikipedia.org/wiki/Geographic_information_system)) created by research and for research. It is leaded by [CNRS](http://www.cnrs.fr/) within the French [Lab-STICC](http://www.lab-sticc.fr/) laboratory ([DECIDE](http://www.labsticc.fr/le-pole-cid/decision-aid-and-knowledge-discovery-decide/) team of Vannes) and licensed under [GPLv3](https://github.com/orbisgis/orbisgis/blob/master/Licenses/license-GPL.txt). OrbisGIS proposes new methods and techniques to model, represent, process and share spatial data, making it easy to monitor geographical territories and manage their evolution.
```
Comment (gpetit) [inline README.md]: [OrbisGIS](http://orbisgis.org/) without spaces

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 In a world ever-increasingly aware of its ecological footprint and the relevance of sustainable development, a systematic approach to evaluating public policies is of paramount importance. Such an approach must take into account relevant environmental, social and economic factors to facilitate efficient decision making and planning. As an integrated modeling platform containing analytical tools for computing various indicators at different spatial and temporal scales, OrbisGIS is already an indispensable instrument for many. Come see what all the buzz is about!
```
Comment (gpetit) [inline README.md]: Just after the dev part, maybe add : 

Users, please consult the on-line documentation on [doc.orbisgis.org](http://doc.orbisgis.org/).

File: README.md (hunk 1 above)
Comment (ebocher) [inline README.md]: It has been fixed

File: README.md (hunk 2 above)
Comment (ebocher) [inline README.md]: yep

Comment (ebocher) [inline README.md]: done

**Pull request merged**